RemoteHealthcare/Bluetooth.cs

```diff
@@ -22,62 +19,24 @@ public static async Task<int> SetConnectionAsync(BLE ble, string device, string
             return errorCode;
         }
 
-        private static void BleHeart_SubscriptionValueChanged(object sender, BLESubscriptionValueChangedEventArgs e)
+        private static void BleHeart_SubscriptionValueChanged(object sender, avansBikeData e)
         {
-            if (e.Data[0] != 0x16)
-                return;
+            if (e.Data[0] != 0x16) { return; }
             Console.WriteLine($"Heartrate: {e.Data[1]} BPM");
         }
 
-        //public class RealBikeData : BikeData
-        //{
-        //    public byte[] Data { get; set; }
-        //    public string ServiceName { get; set; }
-
-        //    public RealBikeData(BLESubscriptionValueChangedEventArgs e)
-        //    {
-        //        this.Data = e.Data;
-        //        this.ServiceName = e.ServiceName;
-        //    }
-
-        //    public RealBikeData()
-        //    {
-        //    }
-        //}
-
-        //private static void BleBike_SubscriptionValueChanged(object sender, BLESubscriptionValueChangedEventArgs e)
-        //{
-        //    RealBikeData realBike = new RealBikeData(e);
-        //    BleBike_SubscriptionValueChanged(realBike, true);
-        //}
-
-        public static void BleBike_SubscriptionValueChanged(avansBikeData e, bool isLogging)
+        public static void BleBike_SubscriptionValueChanged(avansBikeData bikeData)
```

Comment:
Zie [Style Guide](https://github.com/Ti-2-1-RH-A1/remote-healthcare/blob/development/.github/STYLE_GUIDELINES.md#methodes) voor fout bij Methode structuur fout (Danwel hoe methode is opgebouwd of commentaar)